src/dashboard/components/StatsGrid/StatsGrid.tsx

```diff
@@ -0,0 +1,70 @@
+import { Group, Paper, SimpleGrid, Text } from '@mantine/core';
+import {
+  IconUserPlus,
+  IconDiscount2,
+  IconReceipt2,
+  IconCoin,
+  IconArrowUpRight,
+  IconArrowDownRight,
+  IconEqual,
+} from '@tabler/icons-react';
+import classes from './StatsGrid.module.css';
+
+const icons = {
+  user: IconUserPlus,
+  discount: IconDiscount2,
+  receipt: IconReceipt2,
+  coin: IconCoin,
+};
+
+export interface TransactionStatsProps {
+  statistics: {
+    title: string;
+    diff: number;
+    icon: string;
+    value: number;
+  }[];
+}
+
+const StatsGrid: React.FC<TransactionStatsProps> = ({ statistics }) => {
+  const stats = statistics.map((stat) => {
+    const Icon = icons[stat.icon as keyof typeof icons];
```

Comment:
Might be better if there is a enum for `icons` instead of force casting here.

Reply:
done